### DISCUSSION TITLE
Tailwind docs url query search

### DISCUSSION THREAD

**Discussion by Fabian9799** [tailwindlabs/tailwindcss#10278] [Ideas]
Most browsers nowdays allow the user to add custom search engines for the searchbar but only allow query params for the search.
Right now I need to go to the tailwind website and wait until the website is fully loaded until I can start with my search.

YouTube for example uses `?search_query=something` which lets me search youtube right from the address bar
![image](https://user-images.githubusercontent.com/19490891/211391407-543c382f-2b39-49ae-8d7c-d6ecff7696b1.png)

Enabling URL queries would significantly enhance the workflow for myself and many others.

I don't have any experience with Algolia but this is what I found with a quick google search:
https://www.algolia.com/doc/guides/building-search-ui/going-further/routing-urls/js/#basic-urls

**Comment by budivoogt** [comment]
I agree and would love the same!

**Comment by Rromanw34** [comment]
I would like the same.
[DGCustomerFirst](https://dgcustomerfirst.fun/)

**Comment by UnchartedSystems** [comment]
This would be great!

**Comment by t-dub** [comment]
Yes please!  Doing this would also allow us to add the TailwindCSS docs as a "bang" to Kagi or DuckDuckGo
